cats/cats-redis/src/main/java/com/netflix/spinnaker/cats/redis/cluster/ClusteredAgentScheduler.java

```diff
@@ -87,7 +83,8 @@ private void runAgents() {
         activeAgents.putAll(thisRun);
         for (final Map.Entry<String, Long> toRun : thisRun.entrySet()) {
             final AgentExecutionAction exec = agents.get(toRun.getKey());
-            agentExecutionPool.submit(new AgentJob(toRun.getValue(), exec, this));
+            Future s = agentExecutionPool.submit(new AgentJob(toRun.getValue(), exec, this));
+            assert !s.isCancelled();
```

Comment:
would rather see `@SuppressFBWarnings` than the random assert - and need to figure out why that warning shows up only for you folks (I suspect something in the JVM you guys use)